cmd/eip_list.go

```diff
@@ -9,11 +9,12 @@ import (
 )
 
 type eipListItemOutput struct {
-	ID        string   `json:"id"`
-	Zone      string   `json:"zone"`
-	IPAddress string   `json:"ip_address"`
-	Managed   bool     `json:"managed"`
-	Instances []string `json:"instances,omitempty"`
+	Description string   `json:"description"`
```

Comment:
Can you please move the `Description` field after the `IPAddress` one? 😬

Reply:
done